zulip/integrations/bridge_with_irc/irc_mirror_backend.py

```diff
@@ -1,16 +1,16 @@
+import logging
 import multiprocessing as mp
 import sys
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 import irc.bot
 import irc.strings
 from irc.client import Event, ServerConnection, ip_numstr_to_quad
-from irc.client_aio import AioReactor
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
This looks like a temporary debugging measure that should be removed.

Reply:
This is the log when I didn't specify the SASL password to Libera.Chat. This message wouldn't show up unless I set the level to `logging.DEBUG`.
```
DEBUG:irc.client:FROM SERVER: :sodium.libera.chat NOTICE zz_zulip :*** Notice -- SASL authentication to a NickServ account wi
th a verified email address is required to connect from your current network. Please see https://libera.chat/guides/sasl for
configuration assistance.
...
DEBUG:irc.client:FROM SERVER: ERROR :Closing Link: rein.zulipdev.org (SASL authentication to a NickServ account with a verifi
ed email address is required to connect from your current network. Please see https://libera.chat/guides/sasl for configurati
on assistance.)
```

Reply:
Well that’d be a side effect of logging [all data sent from the server](https://github.com/jaraco/irc/blob/v20.3.0/irc/client.py#L335).

Try adding an `on_error` handler to `IRCBot`.

Reply:
I added `on_error` that raises `Exception` for all errors. I'm not sure why there is no default `on_error` on the `IRCBot` that reports the errors.

Currently testing by running the bridge for some period of time, to see that there are errors that are recoverable.